tbtaf/publisher/TBTAFPublisher.py

```diff
@@ -3,17 +3,22 @@
 
 @author: @andresmuro , @andres.alvarado , @mestradago , @rnunezc
 '''
+from __future__ import absolute_import
+from __future__ import print_function
 import datetime
 import os
+from xhtml2pdf import pisa
```

Comment:
Need to document this dependency if not part of the default Python installation if required

Reply:
It's already documented on README and requirements.txt